components/net/src/server.rs

```diff
@@ -171,7 +171,7 @@ pub trait NetIdent {
 
     fn net_ident() -> String {
         let hostname = super::hostname().unwrap();
-        let pid = unsafe { libc::getpid() };
+        let pid = process::current_pid();
```

Comment:
I snuck this change in to this PR because it was quick and related to the work in the `core::os::process` module. This change will allow us to be  portable in the `builder-worker` barring any other Linux specific dependencies we may have ;)